PowerEditor/src/WinControls/WindowsDlg/WindowsDlg.cpp

```diff
@@ -598,6 +613,23 @@ void WindowsDlg::updateColumnNames()
 	lvColumn.pszText = const_cast<TCHAR *>(columnText.c_str());
 	lvColumn.cx = static_cast<int>(SendMessage(_hList, LVM_GETCOLUMNWIDTH, 2, 0));
 	SendMessage(_hList, LVM_SETCOLUMN, 2, LPARAM(&lvColumn));
+
+	columnText = pNativeSpeaker->getAttrNameStr(TEXT("Size"), WD_ROOTNODE, WD_CLMNTYPE);

```

Comment:
@eljefe7000

FWIW, this change won't properly translate "size". At present, the column will be translated to "type". See 13801e3540 for guidance.

Also, I bet the "size" translation can be copied from Styler Dialog's `id="25032"` in every language's translation file. Examples: [English](https://github.com/notepad-plus-plus/notepad-plus-plus/blob/13801e35405841cfb8ff0af65023e341ba3dca34/PowerEditor/installer/nativeLang/english.xml#L496) & [French](https://github.com/notepad-plus-plus/notepad-plus-plus/blob/13801e35405841cfb8ff0af65023e341ba3dca34/PowerEditor/installer/nativeLang/french.xml#L538)

Reply:
@AngryGamer 
Fixed in https://github.com/notepad-plus-plus/notepad-plus-plus/commit/eb6da8195b67a3077310a4d787fd4ac37cd21c76 now.
Thank you for your heads up.

Reply:
Thank you both for your help.